code/game/objects/effects/spawners/faction_spawners.dm

```diff
@@ -0,0 +1,204 @@
+/*
+ * USCM weapons
+ */
+/obj/effect/spawner/random/gun/uscm_primary
+	name = "USCM primary weapon spawner"
+	desc = "spawns USCM primary weapons"
+	spawn_nothing_percentage = 0
+	mags_max = 2
+	mags_min = 1
+	guns = list(
+		/obj/item/weapon/gun/rifle/m41a = /obj/item/ammo_magazine/rifle,
+		/obj/item/weapon/gun/rifle/m41a/tactical = /obj/item/ammo_magazine/rifle,
+		/obj/item/weapon/gun/smg/m39 = /obj/item/ammo_magazine/smg/m39,
+		/obj/item/weapon/gun/smg/m39 = /obj/item/ammo_magazine/smg/m39,
+		/obj/item/weapon/gun/shotgun/pump = /datum/ammo/bullet/shotgun/buckshot
+	)
+
+/obj/effect/spawner/random/gun/uscm_primary/lowchance
+	spawn_nothing_percentage = 80
+	icon_state = "loot_rifle_20"
+
+/obj/effect/spawner/random/gun/uscm_primary/midchance
+	spawn_nothing_percentage = 50
+	icon_state = "loot_rifle_50"
+
+/obj/effect/spawner/random/gun/uscm_primary/highchance
+	spawn_nothing_percentage = 80
+	icon_state = "loot_rifle_80"
```

Comment:
changed the high-change and low-change values to equal each other, I didn't run the code but did you mean to do this?

Reply:
Nope